src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -70,6 +71,7 @@ class HoldRequest extends React.Component {
     if (title) {
       title.focus();
     }
+    if (this.state.redirect) this.setState({ redirect: false });
```

Comment:
Nitpick: Many of us are moving to `prettier` and I think it'll be good to adopt it. This line is valid, but it'll be a multi-line statement after using a linter and prettier.

Reply:
thanks! Not familiar with `prettier` but will read up on it.